config/initializers/assets.rb

```diff
@@ -8,6 +8,6 @@
 
 # Precompile additional assets.
 # application.js, application.css, and all non-JS/CSS in app/assets folder are already added.
-Rails.application.config.assets.precompile += ['about.css.sass', 'registration.css', 'report.css', '*.js', '*.coffee']
+Rails.application.config.assets.precompile += ['registration.css', 'report.css', '*.js', '*.coffee']
```

Comment:
Hound is annoying on this.  I guess it doesn't have a knowledge of rubocop defaults, forcing us to state them explicitly?